precompile/contracts/test_utils/test_precompile.go

```diff
@@ -0,0 +1,84 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package testutils
```

Comment:
Looks like this is a duplicate of `precompile/testutils/test_precompile.go`

Reply:
seems like it wasn't deleted.